### PR TITLE
EN Afin Dialogue 

### PR DESCRIPTION
**Pull request by Drinia**
Translated dialogue (in progress)

Let me know if anything sounds weird. There is another file with the same lines of dialogue (PSO2ENPatchCSV/Dialogue/Afin/np_012100.csv), let me know if those need to be changed as well.

@ Char Amicitia # 0251 on discord (Ship 10)

### CODE REVIEW

File: Dialogue/Afin/np_010030.csv [hunk 1]
```diff
@@ -19,30 +19,30 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
-11_voice_NPC_5003_100#3,"""アークス内で一番えらい人たち……<br>六芒均衡だっけ？\u3000どんなんなんだろ。<br>揃いも揃ってムキムキだったり？"""
+11_voice_NPC_5003_104#2,"""There are some ARCs that are not only good at fighting but gathering info. I wonder if I'm the same..."""
+11_voice_NPC_5003_104#3,"""I knew finding people would be hard but this will take a while. Guess I'll just have to be patient."""
+11_voice_NPC_5003_109#0,"""Another Darker appeared! At least that's what it feels like a lot these days. Maybe it's just me?"""
+11_voice_NPC_5003_101#2,"""Seeing you complete so many client orders is making me happy too. Hehe."""
+11_voice_NPC_5003_108#1,"""I was told by my superiors that “you don't have any courage but you know that”. A bit harsh don't you think?"""
+Afin_15_020#0,"""The thing that's scary is that there are people who think they have courage when they don't. ...Guess they're just reckless."""
+11_voice_NPC_5003_105#2,"""I've been thinking. The Darker have been our enemies for a while now but It hasn't been properly been recorded in history..."""
+11_voice_NPC_5003_104#4,"""ARCs are...loved by everyone. But I never thought it would be this tiring...ha-."""
```
Comment (SynthSy) [inline Dialogue/Afin/np_010030.csv]: first off, it's ARKS not ARCs, please see https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki/Story

Comment (Drinia) [reply Dialogue/Afin/np_010030.csv]: I realized that a short while after submitting it. Thank you for the link, I shall bookmark it.

File: Dialogue/Afin/np_010030.csv [hunk 2]
```diff
@@ -19,44 +19,44 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
-11_voice_NPC_5003_100#3,"""アークス内で一番えらい人たち……<br>六芒均衡だっけ？\u3000どんなんなんだろ。<br>揃いも揃ってムキムキだったり？"""
+11_voice_NPC_5003_105#1,"""We can change classes but the higher ranks have already chosen theirs. Why did it change...?"""
+11_voice_NPC_5003_104#2,"""There are some ARKS that are not only good at fighting but gathering info. I wonder if I'm the same..."""
+11_voice_NPC_5003_104#3,"""I knew finding people would be hard but this will take a while. Guess I'll just have to be patient."""
+11_voice_NPC_5003_109#0,"""Another Darker appeared! At least that's what it feels like a lot these days. Maybe it's just me?"""
+11_voice_NPC_5003_101#2,"""Seeing you complete so many client orders is making me happy too. Hehe."""
+11_voice_NPC_5003_108#1,"""I was told by my superiors that “you don't have any courage but you know that”. A bit harsh don't you think?"""
+Afin_15_020#0,"""The thing that's scary is that there are people who think they have courage when they don't. ...Guess they're just reckless."""
+11_voice_NPC_5003_105#2,"""I've been thinking. The Darker have been our enemies for a while now but It hasn't been properly been recorded in history..."""
```
Comment (SynthSy) [inline Dialogue/Afin/np_010030.csv]: The Darkers, always plural, unless you're referring to a singular thing

File: Dialogue/Afin/np_010030.csv [hunk 3]
```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
-11_voice_NPC_5003_105#3,"""…………<br>あっ、わ、悪い相棒。<br>ちょっと考え事してた。"""
-Afin_mb12_020#0,"""考えても何も解決しないって<br>そんなことわかってるんだけど……<br>おれ、切り替えが下手でさ。"""
-11_voice_NPC_5003_105#4,"""…………<br>あっ、わ、悪い相棒。<br>ちょっと考え事してた。"""
-Afin_mb13_020#0,"""考えても何も解決しないって<br>そんなことわかってるんだけど……<br>おれ、切り替えが下手でさ。"""
-11_voice_NPC_5003_104#6,"""惑星リリーパの採掘場跡、か。<br>……何かありそうな気がする。"""
+11_voice_NPC_5003_109#1,"""It's weird how the ARKS ship can be attacked directly and no one panics. I guess it happens too often for people to get upset."""
+Afin_19_020#0,"""...I feel like there is something wrong with you if you think that that is normal...。"""
+11_voice_NPC_5003_104#5,"""I didn't join ARKS because I wanted to protect someone."""
+Afin_20_020#0,"""Like I thought, if we get attacked here I end up thinking I want to protect everyone. It's weird to think about."""
+11_voice_NPC_03_104#0,"""Ever since the Dark Falz has been revived, everyone's been on edge."""
```
Comment (Drinia) [inline Dialogue/Afin/np_010030.csv]: '...the Dark Falz' or just 'since Dark Falz'?

Comment (alama) [reply Dialogue/Afin/np_010030.csv]: There is a misspelling:
 seperated or separated?

File: Dialogue/Afin/np_010030.csv [hunk 4]
```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
+11_voice_NPC_03_104#1,"""Well, I think it's good to let off a little steam in the ARKS Battle Tournament every once and a while."""
+Afin_mb11_020#0,"""Looks like a lot of people are joining in...what should I do?"""
+11_voice_NPC_5003_105#3,"""...Ah, my bad partner. I was spacing out."""
+Afin_mb12_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
+11_voice_NPC_5003_105#4,"""...Ah, my bad partner. I was spacing out."""
+Afin_mb13_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
+11_voice_NPC_5003_104#6,"""The Quarry on Planet Lillipa....I feel like there's something there."""
 Afin_mb14_020#0,"""あ、別にお宝一攫千金、って<br>ことじゃないからな？<br>そんなの取り尽くされちゃってるだろ。"""
-11_voice_NPC_5003_104#7,"""……ほんと、冗談みたいだよな。<br>生き別れた姉がダークファルスって<br>どうなってんだよ……"""
+11_voice_NPC_5003_104#7,"""This...seems like a joke. My sister that I was seperated from is the Dark Falz...what's happening..."""
```
Comment (alama) [inline Dialogue/Afin/np_010030.csv]: There is a misspelling:
seperated or separated?

Comment (Drinia) [reply Dialogue/Afin/np_010030.csv]: I know English I swear... it's separated, will fix now.

**Comment by alama** [comment]
Now, since there been a PSO2 update, we need you to [update your EN branch](https://github.com/Drinia/PSO2ENPatchCSV/pull/5)

File: Dialogue/Afin/np_010030.csv (hunk 3 above)
Comment (SynthSy) [reply Dialogue/Afin/np_010030.csv]: since Dark Falz

File: Dialogue/Afin/np_010030.csv [hunk 5]
```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
+11_voice_NPC_5003_105#4,"""...Ah, my bad partner. I was spacing out."""
+Afin_mb13_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
+11_voice_NPC_5003_104#6,"""The Quarry on Planet Lillipa....I feel like there's something there."""
 Afin_mb14_020#0,"""あ、別にお宝一攫千金、って<br>ことじゃないからな？<br>そんなの取り尽くされちゃってるだろ。"""
-11_voice_NPC_5003_104#7,"""……ほんと、冗談みたいだよな。<br>生き別れた姉がダークファルスって<br>どうなってんだよ……"""
+11_voice_NPC_5003_104#7,"""This...seems like a joke. My sister that I was separated from is the Dark Falz...what's happening..."""
 Afin_mb15_020#0,"""これならまだ、死別ってほうが<br>諦めがつくけど…………"""
 Afin_mb15_030#0,"""……いや、生きててくれたんなら<br>なんとかなる。<br>きっと、なんとかなるはずだ。"""
-Afin_mb16_010#0,"""……ほんと、冗談みたいだよな。<br>生き別れた姉がダークファルスって<br>どうなってんだよ……"""
+Afin_mb16_010#0,"""This...seems like a joke. My sister that I was seperated from is the Dark Falz...what's happening..."""
```
Comment (alama) [inline Dialogue/Afin/np_010030.csv]: There is a misspelling:
seperated or separated?

File: Dialogue/Afin/np_012100.csv [hunk 6]
```diff
@@ -19,73 +19,73 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
...
+11_voice_NPC_5210_105#4,"""...Ah, my bad partner. I was spacing out."""
+Afin_mb13_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
+11_voice_NPC_5210_104#6,"""The Quarry on Planet Lillipa....I feel like there's something there."""
 Afin_mb14_020#0,"""あ、別にお宝一攫千金、って<br>ことじゃないからな？<br>そんなの取り尽くされちゃってるだろ。"""
-11_voice_NPC_5210_104#7,"""……ほんと、冗談みたいだよな。<br>生き別れた姉がダークファルスって<br>どうなってんだよ……"""
+11_voice_NPC_5210_104#7,"""This...seems like a joke. My sister that I was separated from is the Dark Falz...what's happening..."""
 Afin_mb15_020#0,"""これならまだ、死別ってほうが<br>諦めがつくけど…………"""
 Afin_mb15_030#0,"""……いや、生きててくれたんなら<br>なんとかなる。<br>きっと、なんとかなるはずだ。"""
-Afin_mb16_010#0,"""……ほんと、冗談みたいだよな。<br>生き別れた姉がダークファルスって<br>どうなってんだよ……"""
+Afin_mb16_010#0,"""This...seems like a joke. My sister that I was seperated from is the Dark Falz...what's happening..."""
```
Comment (alama) [inline Dialogue/Afin/np_012100.csv]: There is a misspelling:
seperated or separated?

**Pull request merged**